patterns/1-initial/release-process.md

```diff
@@ -0,0 +1,48 @@
+## Title
+
+Standard Release Process/Published Artifacts
+
+## Patlet
+
+Teams may be reluctant to use InnerSource projects that they are unfamiliar with when there is no clear release process apparent in the repository.
+Providing clear release notes and a published artifact (binary, docker image, jar, etc) gives people confidence you are publishing a quality product.
+
+## Problem
+
+The common code in the shared repository doesn't have a published artifact or publicly viewable release process.
+
+## Context
+
+It is common practice in Open Source projects to have releases, with release notes documenting breaking changes,
+new features, etc along with either a published binary or link to a Docker image. This practice may not be as
+transparent or well documented/visible for InnerSource projects, modules, etc. Providing robust release notes
+along with a published artifact that is the result of a clearly document and visible release process builds trust and confidence in your project.
+
+## Forces
+
+- Difficult for organizations that don't have a central CI/CD system
+- Adds the burden of publishing release notes
+- No internal location to host artifacts
+
+## Solution
+
+- CI/Delivery Pipeline is located within your repo to build artifacts
```

Comment:
```suggestion
- CI/Delivery Pipeline is located within your repo to build artifacts (binary, docker image, jar, etc)
```

Adding examples here to make sure that people know what is meant by artifacts in this context.